beacon-chain/rpc/beaconv1/config.go

```diff
@@ -23,5 +30,21 @@ func (bs *Server) GetSpec(ctx context.Context, req *ptypes.Empty) (*ethpb.SpecRe
 
 // GetDepositContract retrieves deposit contract address and genesis fork version.
 func (bs *Server) GetDepositContract(ctx context.Context, req *ptypes.Empty) (*ethpb.DepositContractResponse, error) {
-	return nil, errors.New("unimplemented")
+	ctx, span := trace.StartSpan(ctx, "beaconv1.GetDepositContract")
+	defer span.End()
+
+	buf := bytes.NewBuffer(params.BeaconConfig().GenesisForkVersion)
+	chainId, err := binary.ReadUvarint(buf)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "could not obtain genesis fork version: %v", err)
+	}
```

Comment:
I did not test the error case as this can only happen when our fork version is completely broken.